### PR TITLE
docs(microservices): add instruction to use transport in MessagePattern

### PR DESCRIPTION
**Pull request by barbarosh**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/nestjs/docs.nestjs.com/blob/master/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number:  https://github.com/nestjs/nest/issues/4769

## What is the new behavior?
Add code example and instruction how to use transport in MessagePattern decorator

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

## Other information

### CODE REVIEW

**Comment by kamilmysliwiec** [comment]
Thanks for the PR! I believe that we should rather add it to the Hybrid Applications chapter instead. Adding more content to the "Basics" section that is irrelevant to many developers might just make this chapter unnecessarily complicated. Do you have any thoughts here @johnbiundo?

**Comment by johnbiundo** [comment]
I agree with @kamilmysliwiec's comment about moving to the hybrid chapter.

**Comment by barbarosh** [comment]
@kamilmysliwiec @johnbiundo Ok I will move

**Comment by kamilmysliwiec** [comment]
@barbarosh do you have some time to look into this by chance? 🏓

**Comment by barbarosh** [comment]
> @barbarosh do you have some time to look into this by chance? 🏓  

Yes I will do it on this weekend

**Comment by barbarosh** [comment]
> @barbarosh do you have some time to look into this by chance? 🏓

on this week I will do it 😅

**Comment by barbarosh** [comment]
@kamilmysliwiec Updated PR

**Comment by kamilmysliwiec** [comment]
Would you be able to review this PR @jmcdo29? :) (grammar-wise etc) Apologies for pinging you in so many places! Trying to clean up some of the remaining, opened PRs

File: content/faq/hybrid-application.md [hunk 1]
```diff
@@ -35,6 +35,35 @@ await app.startAllMicroservicesAsync();
 await app.listen(3001);
 ```
 
+For bind `@MessagePattern()` only to one transport of microservices in hybrid application with multiple microservices can use second argument which should get value from `Transport` enum
```
Comment (jmcdo29) [inline content/faq/hybrid-application.md:38]: For bind...

To

To bind...

Comment (jmcdo29) [inline content/faq/hybrid-application.md:38]: `with multiple microservices can use`

To

`with multiple microservices we can use`

Comment (jmcdo29) [inline content/faq/hybrid-application.md:38]: `can use second argument`

To

`can use the second argument`

Comment (jmcdo29) [inline content/faq/hybrid-application.md:38]: `only to one`

To

`to only one`

**Review by jmcdo29 — CHANGES_REQUESTED** [review]
Made some small grammar related comments

File: content/faq/hybrid-application.md (hunk 1 above)
Comment (kamilmysliwiec) [inline content/faq/hybrid-application.md:38]:
```suggestion
To bind `@MessagePattern()` to only one transport strategy (for example, MQTT) in a hybrid application with multiple microservices, we can pass the second argument of type `Transport` which is an enum with all the built-in transport strategies defined.
```

**Comment by kamilmysliwiec** [comment]
Thanks!

**Pull request merged**